biddings/views.py

```diff
@@ -50,4 +55,30 @@ def get(self, request):
             }for product in sell_products],
         }
 
-        return JsonResponse({"result" : result}, status=200)
\ No newline at end of file
+        return JsonResponse({"result" : result}, status=200)
+
+class NewBiddingView(View):
+    @login_decorator
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+
+            user     = request.user
+            is_buyer = data['is_buyer']
+            price    = data['offered_price']
+            product  = data['id']
+            size     = data['size']
+
+            size_id      = Size.objects.get(name = size).id
+            product_size = ProductSize.objects.get(size_id = size_id, product_id = product).id
```

Comment:
```suggestion
            if ProductSize.objects.filter(size__name = size, product_id = product).exists():
                return JsonResponse({"message" :  "product_size does not exist"}, status=204)
                
            product_size = ProductSize.objects.get(size__name = size, product_id = product).id
```
1. objects.get을 사용하기 전에 예외처리 추가해 주세요.
2. id를 찾는 과정 없이 size의 이름을 가지고 바로 데이터를 가져올 수 있습니다.